plutus-core/plutus-core/src/PlutusCore/Error.hs

```diff
@@ -202,6 +216,13 @@ instance HasErrorCode ParserError where
     errorCode UnknownBuiltinType {}     = ErrorCode 8
     errorCode BuiltinTypeNotAStar {}    = ErrorCode 51
 
+instance HasErrorCode (ParseErrorBundle T.Text ParserError) where
+    errorCode (ParseErrorBundle (NE.head -> (FancyError _ (Set.findMin -> (ErrorCustom InvalidBuiltinConstant {})))) _) = ErrorCode 10
+    errorCode (ParseErrorBundle (NE.head -> (FancyError _ (Set.findMin -> (ErrorCustom UnknownBuiltinFunction {})))) _) = ErrorCode 9
+    errorCode (ParseErrorBundle (NE.head -> (FancyError _ (Set.findMin -> (ErrorCustom UnknownBuiltinType {})))) _)     = ErrorCode 8
+    errorCode (ParseErrorBundle (NE.head -> (FancyError _ (Set.findMin -> (ErrorCustom BuiltinTypeNotAStar {})))) _)    = ErrorCode 51
+    errorCode _ = error "errorCode: Not a valid parser error."
```

Comment:
I don't think we should do this. 
- It's partial
- `ParseErrorBundle`s contain multiple errors!

I think we should just assign one error code for "parse error".

Reply:
OMG I've looked at this code for too long clearly! >.< I was trying to make it match the existing error codes for the `ParserError`'s, and I think I should still do that so that we have more info. But yeah I'll add a code for all other parse errors.